assets/css/editor.scss

```diff
@@ -40,7 +40,7 @@
 		margin: -$gap 0 $gap;
 	}
 
-	& + .wc-block-base-control-notice:last-child {
+	+ .wc-block-base-control-notice:last-child {
```

Comment:
Why this change? It seems the intention is changed? 🤔

Reply:
This is actually the same and was marked as an error by css lint.  ref. https://css-tricks.com/the-sass-ampersand/#aa-using-the-with-and

Reply:
You're right!